src/components/app/flyouts/settings/platformSettings.js

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+
+import { Btn, FormControl, Indicator, Svg, FileInput } from 'components/shared';
+import _ from 'lodash';
+import Flyout from 'components/shared/flyout';
+
+import './platformSettings.css';
+import { svgs } from 'utilities';
+const Section = Flyout.Section;
+
+class PlatformSettings extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentLogo: this.props.logo,
+      currentApplicationName: this.props.name,
+      stillInitializing: false,
+      edit: false,
+      previewLogo: this.props.logo,
+      newLogoName: undefined,
```

Comment:
newLogoName: null

Reply:
The code guide says "Use undefined, not null"